### PR TITLE
add sound option

### PR DESCRIPTION
**Pull request by thiagopnts**
This adds a `sound` option to enable a sound on the notification.

### CODE REVIEW

File: notification_center.py [hunk 1]
```diff
@@ -26,15 +29,16 @@
 weechat.hook_print('', 'irc_privmsg', '', 1, 'notify', '')
 
 def notify(data, buffer, date, tags, displayed, highlight, prefix, message):
-	if weechat.config_get_plugin('show_highlights') == 'on' and highlight == '1':
```
Comment (sindresorhus) [inline notification_center.py]: Why did you change `highlight == '1'`?

File: notification_center.py [hunk 2]
```diff
@@ -7,7 +9,7 @@
 
 SCRIPT_NAME = 'notification_center'
 SCRIPT_AUTHOR = 'Sindre Sorhus <sindresorhus@gmail.com>'
-SCRIPT_VERSION = '0.2.0'
+SCRIPT_VERSION = '0.2.1'
```
Comment (sindresorhus) [inline notification_center.py]: Don't bump version in other peoples projects. It's also the wrong version.

File: notification_center.py [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # https://github.com/sindresorhus/weechat-notification-center
 # Requires `pip install pync`
+# Author: Sindre Sorhus <sindresorhus@gmail.com>
+# License: MIT
```
Comment (sindresorhus) [inline notification_center.py]: This is moot. It's already below.

File: notification_center.py [hunk 4]
```diff
@@ -26,15 +29,16 @@
 weechat.hook_print('', 'irc_privmsg', '', 1, 'notify', '')
 
 def notify(data, buffer, date, tags, displayed, highlight, prefix, message):
-	if weechat.config_get_plugin('show_highlights') == 'on' and highlight == '1':
+        sound = 'Pong' if weechat.config_get_plugin('sound') == 'on' else lambda:_
```
Comment (sindresorhus) [inline notification_center.py]: What does `lambda:_` mean? I've never used that before.

**Comment by sindresorhus** [comment]
Did you test this with latest version of Weechat? I don't use Weechat anymore, so won't be able to test.

File: notification_center.py (hunk 2 above)
Comment (thiagopnts) [inline notification_center.py]: Opss, sorry. I didn't notice this changes, I've being using this for quite sometime and realized it could be useful for other people too. I'll remove the unnecessary changes and squash.

File: notification_center.py (hunk 4 above)
Comment (thiagopnts) [inline notification_center.py]: This is just a dirty hack because the `Notifier.notify` call fallbacks to the default sound even if you pass `None`or `''` to the sound parameter, passing a lambda makes no sound.

Comment (sindresorhus) [inline notification_center.py]: Ok, thanks for the explanation. Makes sense.

**Comment by thiagopnts** [comment]
I just updated the PR, I use this daily so if anyone find something wrong ping me

**Comment by sindresorhus** [comment]
Thanks. I sent in an update to the WeeChat script thingy.
